apps/coscrad-frontend/src/components/Home/Home.spec.tsx

```diff
@@ -1,15 +1,37 @@
+import { screen } from '@testing-library/react';
 import { MemoryRouter } from 'react-router-dom';
+import { getDummyConfigurableContent } from '../../app/App.spec';
 import { renderWithProviders } from '../../utils/test-utils/renderWithProviders';
 
-import Home from './Home';
+import { Home } from './Home';
 
 describe('Home', () => {
-    it('should render successfully', () => {
+    it('should load configurable content', () => {
+        const dummyConfigurableContent = getDummyConfigurableContent();
+        const siteDescription = 'Describe my site here';
+        const siteHomeImageUrl = 'https://mysite.com/homeimg.png';
+        const myConfigurableContent = {
+            ...dummyConfigurableContent,
+            siteDescription,
+            siteHomeImageUrl,
+        };
+
         const { baseElement } = renderWithProviders(
             <MemoryRouter>
                 <Home />
-            </MemoryRouter>
+            </MemoryRouter>,
+            {
+                contentConfig: myConfigurableContent,
+            }
         );
-        expect(baseElement).toBeTruthy();
+
+        const searchPattern = new RegExp(siteDescription);
+        const screenRes = screen.getByText(searchPattern);
+
+        const logo = baseElement.querySelector('img');
+
+        // Split into two tests?  Use a helper function for loading the configurable content?
```

Comment:
Do this.

Reply:
Maybe we could pair on this so that I can see the best way to do this.  E.g., is it a `beforeAll()`?